packages/vc-api/src/controllers/issuer-controller.ts

```diff
@@ -16,14 +16,14 @@ export const issueCredential = async (req: RequestWithAgent, res: Response) => {
     const payload = {
       ...DEFAULT_CONFIG,
       credential: req.body.credential,
-      proofFormat: req.body.options.proofFormat ?? DEFAULT_CONFIG.proofFormat,
+      proofFormat: req.body?.options?.proofFormat ?? DEFAULT_CONFIG.proofFormat,
```

Comment:
I noticed that the `options` field in the request payload is structured in the specifications as follows:

- mandatoryPointers [array]
- credentialId [string]

If we add an additional field like `proofFormat`, would that be correct and compliant with the specifications?

Here’s the link to the specs: https://w3c-ccg.github.io/vc-api/#issue-credential

Reply:
The options field is used to add the information when making the request, it should be fine and comply with the spec if we don't use the defined field for another purpose.